tableview/src/main/java/com/evrencoskun/tableview/handler/ScrollHandler.java

```diff
@@ -93,14 +103,37 @@ private void scrollCellHorizontally(int columnPosition, int offset) {
 
                 columnLayoutManager.scrollToPositionWithOffset(columnPosition, offset);
             }
-
         }
     }
 
+    public void scrollToNextField() {
+        final CellRecyclerView child = (CellRecyclerView) mCellLayoutManager.getChildAt(mTableView.getSelectedRow());
+        child.post(new Runnable() {
+            @Override
+            public void run() {
+                child.scrollBy(300, 0);
```

Comment:
can you use smooth scroll??

Reply:
Nope. Smooth scroll is messing with all the scroll listeners